### PR TITLE
completed html-web-page exercise

### PR DESCRIPTION
**Pull request by 16hsuisaac**
html-web-page

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      <select name="plan">
+        <option value="">Select your plan:</option>
+        <option value="Basic">Basic Plan</option>
+        <option value="Deluxe">Deluxe Plan</option>
+        <option value="Jelly">Jelly Giant Plan</option>
+      </select>
+    </label>
+    <label>
+      <button>Submit</button>
+    </label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:84]: The label elements here are unnecessary, since there is no text labelling these elements. They can be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One minor note:

**Comment by 16hsuisaac** [comment]
html-web-page

**Review by codingcodymiller — APPROVED** [review]
Nice work!!!!

**Pull request merged**